packages/components/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { styled, theme } from "@localyze-pluto/theme";
+import { Root, Indicator, ProgressProps } from "@radix-ui/react-progress";
+
+const StyledProgress = styled.div`
+  background-color: ${theme.colors.colorBackgroundWeak};
+  border-radius: ${theme.radii.borderRadius10};
+  height: 6px;
+  overflow: hidden;
+  position: relative;
+`;
+
+const StyledIndicator = styled.div`
+  height: 100%;
+  width: 100%;
+  transition: transform 660ms cubic-bezier(0.65, 0, 0.35, 1);
+  border-radius: ${theme.radii.borderRadius10};
+  background-color: ${(props: { color: string }) =>
+    props.color === "green"
+      ? theme.colors.colorIconSuccess
+      : theme.colors.colorBackgroundPrimaryWeak};
+`;
+
+export interface ProgressBarProps extends Omit<ProgressProps, "value"> {
+  /** The progress value. */
+  value: number;
+  /** The color of the progress bar. */
+  color?: "blue" | "green";
```

Comment:
Should the color be controlled inside the component? I understood that the bar turns green when it is 100%.

Reply:
We have some [additional design on Figma](https://www.figma.com/file/0KtV7fzjug9hL3XOLzqn1l/%5B-WIP-%5D-%F0%9F%AA%90-Pluto-Components?node-id=387%3A2956) that shows the 100% as blue. So, I decided to remove the prop color for this first version and add it if necessary soon.